infrasim/ipmi.py

```diff
@@ -13,7 +13,7 @@ def get_ipmi():
 def start_ipmi(node):
     ipmi_cmd = get_ipmi()
     ipmi_start_cmd = "{0} -c /etc/infrasim/vbmc.conf" \
-                    " -f /usr/local/etc/infrasim/{1}/{1}.emu -n -s /var/tmp &".format(ipmi_cmd, node)
+                    " -f /usr/local/etc/infrasim/{1}/{1}.emu -n -s /var/tmp 2>1 1>/dev/null &".format(ipmi_cmd, node)
```

Comment:
if redirect command output to null device, how to check what happened for the command in case of the command failed?